server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@prvbharadwaj1731 you should be able to share the aesd_ioctl.h file content with the char driver implementation.